app/src/components/Login.tsx

```
@@ -160,39 +169,39 @@ export const Login = () => {
-                        className="text-slate-500 hover:text-indigo-600 font-bold text-sm transition-colors group"
+                        className="text-indigo-200/50 hover:text-white font-bold text-sm transition-all group flex items-center justify-center gap-2 mx-auto"
                     >
                         {isSignUp ? (
                             <>
-                                すでにアカウントをお持ちですか？ <span className="text-indigo-600 group-hover:underline">ログイン</span>
+                                すでにアカウントをお持ちですか？ <span className="text-indigo-400 group-hover:text-indigo-300 underline decoration-indigo-400/30 underline-offset-4">ログイン</span>
                             </>
                         ) : (
                             <>
-                                はじめての方はこちら <span className="text-indigo-600 group-hover:underline">新規登録 (招待制)</span>
+                                初めてのご利用ですか？ <span className="text-indigo-400 group-hover:text-indigo-300 underline decoration-indigo-400/30 underline-offset-4">新規登録はこちら</span>
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)